go.mod

```diff
@@ -18,36 +16,34 @@ require (
 	github.com/go-test/deep v1.0.3
 	github.com/gogo/protobuf v1.2.1 // indirect
 	github.com/golang/groupcache v0.0.0-20180924190550-6f2cf27854a4 // indirect
+	github.com/golang/mock v1.3.1
```

Comment:
We already have counterfeiter (and `bin/gen-fakes`), which is used in other bosh projects, too. 
Besides being more popular, does mockgen have any advantages in your opinion?